docs/security/trust/trust_sandbox.md

```diff
@@ -130,13 +130,13 @@ So, you'll need an entry for both the servers in your local `/etc/hosts` file.
 In this step, you get the source code for your notary and registry services.
 Then, you'll use Docker Compose to build and start them on your local system.
 
-1. Change to back to the root of your  `notarysandbox` directory.
+1. Change to back to the root of your  `trustsandbox` directory.
 
-        $ cd notarysandbox
+        $ cd trustsandbox
 
 2. Clone the `notary` project.
 
-          $ git clone -b trust-sandbox https://github.com/docker/notary.git
+          $ git clone https://github.com/docker/notary.git
 
 3. Clone the `distribution` project.
```

Comment:
Done, thanks for spotting this!  I think I had previously removed this but messed up a stash conflict resolution.